src/Authentication/Authentication/Cmdlets/InvokeMgGraphRequest.cs

```diff
@@ -1090,9 +1090,13 @@ private string QualifyFilePath(string path)
 
         /// <summary>
         ///     Resets GraphSession environment back to its original state.
+        ///     Original state can remain to be the previous state defined by the user, 
+        ///     or the default global state. User defined state can be removed by calling,
+        ///     Remove-MgGraph Command
```

Comment:
We don't have a command called Remove-MgGraph. Did you mean `Disconnect-MgGraph`?

Reply:
@peombwa I meant Remove-MgEnvironment